packages/belos/tpetra/test/BlockGmres/CMakeLists.txt

```diff
@@ -47,6 +39,14 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   ARGS "--verbose --filename=orsirr1.hb"
   STANDARD_PASS_OUTPUT
   )
+ENDIF()
```

Comment:
@ndellingwood shouldn't the test be declared before the `ENDIF()` statement if it requires Triutils to be ON?

Reply:
@ndellingwood I have the same question.  ;)

Reply:
@hkthorn @lucbv the diff looks weird - the change I made was to move the `test_resolve_gmres_hb.cpp` and `test_pseudo_gmres_hb.cpp` into the `IF (${PACKAGE_NAME}_ENABLE_Triutils)` block of code - these were the tests triggering the error without Triutils.
The diff makes it look like I shifted the `ENDIF()` (corresponding to `IF (${PACKAGE_NAME}_ENABLE_Triutils)`) down 18 lines and shifted the `test_bl_gmres_hb.cpp` test out of the block. Effectively the same thing, but looks misleading from the way I made the actual changes.
Please view the full file for confirmation, the git diff obscures the intent of the changes. FWIW I tested this locally while working on the fix and it resolves the issue